tools/run_tests/run_tests.py

```diff
@@ -1526,7 +1526,7 @@ def _build_and_run(check_cancelled,
     '--compiler',
     choices=[
         'default',
-        'gcc4.9',
+        'gcc5',
```

Comment:
It may be worth labeling this `gcc5.1` as well, on the off chance we need to support versions like "gcc >= 5.3" later on. It's unlikely, maybe, but it would also be consistent with the `gcc10.2` argument below.

Reply:
I thought about this but I rather decided not to because gcc changed the way they use the version. From gcc 5, they started to bump the major version every year and use minor versions as a patch version. Therefore, gcc 5.1 and 5.5 aren't that different. 5.5 got the bug fixes only (e.g. [gcc 5 changes](https://gcc.gnu.org/gcc-5/changes.html)) so I decided to drop the minor version from the name. Theoretically, passing all tests with gcc 5.5 doesn't promise that it will pass with gcc 5.1 but it's way less likely to have that case compared to the case of gcc 4.x.

Reply:
Got it, thanks. TIL how gcc's version numbering scheme works https://gcc.gnu.org/develop.html#:~:text=Version%20Numbering%20Scheme%20for%20GCC%205%20and%20Up